tox.ini

```diff
@@ -7,3 +8,8 @@ deps =
     pre-commit
 commands =
     pre-commit run --all-files
+
+[testenv:unittests]
+deps =
+commands =
+    poetry run pytest webhook_server_container/tests/test_github_api.py
```

Comment:
tests failed with error:
`unittests: failed with poetry is not allowed, use allowlist_externals to allow it`

Reply:
Still failed, now with `ModuleNotFoundError: No module named 'pytest'`

pytest should be add to `pyproject.toml` under `unittests` group